src/api/migraphx.py

```diff
@@ -220,6 +220,27 @@ def module(h):
              returns='migraphx::instruction_ref')
 
 
+@auto_handle()
+def value(h):
+    h.constructor('create')
+    h.method('is_null', returns='bool', const=True)
+    h.method('get_key', returns='const std::string', const=True)
+    cpp_types = ['int64_t', 'uint64_t', 'double', 'std::string', 'bool']
+    vt = ['int64', 'uint64', 'float', 'string', 'bool']
+    for vt, cpp_type in zip(vt, cpp_types):
+        if (vt == 'string'):
+            h.constructor('create_' + vt, api.params(i='const char*'))
+            h.constructor('create_' + vt + '_with_key',
+                          api.params(pkey='const char*', i='const char*'))
+        else:
+            h.constructor('create_' + vt, api.params(i=cpp_type))
+            h.constructor('create_' + vt + '_with_key',
+                          api.params(pkey='const char*', i=cpp_type))
+        h.method('if_' + vt, returns='const ' + cpp_type + '*', const=True)
+        h.method('is_' + vt, returns='bool', const=True)
+        h.method('get_' + vt, returns=cpp_type, const=True)
```

Comment:
Actually, I dont think we should add the get methods. Instead lets just add `to_string`, `to_int64`, `to_uint64`, and `to_double` method by calling `v.to<T>()`. Even for now, you could just add `to_string`, which should be enough for unit tests.